src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/data/VisitDto.kt

```diff
@@ -23,6 +23,10 @@ data class VisitDto(
   val visitStatus: String,
   @Schema(description = "Visit Status Description", example = "Reserved", required = true)
   val visitStatusDescription: String,
+  @Schema(description = "Visit Restriction", example = "OPEN", required = true)
+  val visitRestriction: String,
```

Comment:
why is this String why not VisitRestriction

Reply:
Returns the enum name and a description - i dont think both are really needed, but its the way it been designed.